cloudwatch_metrics_collector/include/cloudwatch_metrics_collector/metrics_collector.hpp

```diff
@@ -45,7 +47,15 @@ class MetricsCollector : public Service
    * @param metric_list_msg
    * @return the number of metrics successfully batched
    */
-  int RecordMetrics(const ros_monitoring_msgs::msg::MetricList::UniquePtr metric_list_msg);
+  int RecordMetrics(ros_monitoring_msgs::msg::MetricList::UniquePtr metric_list_msg);
```

Comment:
Why remove the const qualifier?

Reply:
Being a `unique_ptr`, this means ownership of the message is being passed to `RecordMetrics()`. Being `const` here just means that `RecordMetrics()` can't pass ownership of the message to anywhere else. Passing the `unique_ptr` as `const` here is inconsequential in this case, but also unconventional.